translib/transformer/xfmr_event.go

```diff
@@ -0,0 +1,398 @@
+//////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Dell, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//////////////////////////////////////////////////////////////////////////
+
+package transformer
+
+import (
+	"encoding/json"
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Azure/sonic-mgmt-common/translib/db"
+	"github.com/Azure/sonic-mgmt-common/translib/utils"
+	log "github.com/golang/glog"
+)
+
+func init() {
+	XlateFuncBind("DbToYang_event_severity_xfmr", DbToYang_event_severity_xfmr)
+	XlateFuncBind("DbToYang_alarm_severity_xfmr", DbToYang_alarm_severity_xfmr)
+	XlateFuncBind("rpc_unacknowledge_alarms", rpc_unacknowledge_alarms)
+	XlateFuncBind("rpc_acknowledge_alarms", rpc_acknowledge_alarms)
+	XlateFuncBind("rpc_get_events", rpc_get_events)
+	XlateFuncBind("rpc_get_alarms", rpc_get_alarms)
+	XlateFuncBind("YangToDb_event_stats_key_xfmr", YangToDb_event_stats_key_xfmr)
```

Comment:
Why do we need event_stats_key_xfmr if the two functions does not do anything? Where is this defined in annotation file?